test/fixtures/import.mjs

```diff
@@ -1,3 +1,3 @@
-import foo from './export'
+import foo from './export.mjs'
```

Comment:
In Node.js v12, `import` requires explicit file extensions unless `--es-module-specifier-resolution` flag is set to `node`.